packages/salesforcedx-vscode-core/src/orgBrowser/metadataType.ts

```diff
@@ -138,31 +81,27 @@ export type MetadataObject = {
   xmlName: string;
 };
 
-export function buildTypesList(metadataTypesPath: string): string[] {
-  try {
-    const fileData = JSON.parse(fs.readFileSync(metadataTypesPath, 'utf8'));
-    const metadataObjects = fileData.metadataObjects as MetadataObject[];
-    const metadataTypes = [];
-    for (const metadataObject of metadataObjects) {
-      if (!isNullOrUndefined(metadataObject.xmlName)) {
-        metadataTypes.push(metadataObject.xmlName);
-      }
+export function buildTypesList(
```

Comment:
This method should return a string array in alphabetical order. Currently the types are shown in an order that's difficult to navigate.
![Screen Shot 2019-06-24 at 7 12 05 PM](https://user-images.githubusercontent.com/1041842/60064101-30e36780-96b4-11e9-8e61-91d397a1ff44.png)

Reply:
Icons?
![Screen Shot 2019-06-25 at 2 51 41 PM](https://user-images.githubusercontent.com/10779873/60136379-e49e3300-9758-11e9-86e3-0fcbd32304a5.png)

No Icons?
![Screen Shot 2019-06-25 at 2 52 30 PM](https://user-images.githubusercontent.com/10779873/60136392-ecf66e00-9758-11e9-8ffc-0ad608548a05.png)

@ntotten

Reply:
I personally like the icons. Once we expand to "source format" i think it would be good to use the same icons as the file explorer for xml, js, html, cls, etc.

Reply:
One thought, in the file explorer there is an option to show icons or not. Maybe we can use that same setting?

![image](https://user-images.githubusercontent.com/282782/60218237-bdb92d00-983c-11e9-9de8-647fd9c633cf.png)

Reply:
The icons look good to me too, I think we can add them in a future story. I'll log a work item for the icons with the option of exploring that setting ^^